### DISCUSSION TITLE
Advanced Settings WS7200

### DISCUSSION THREAD

**Discussion by edinhodias7** [vmakeev/huawei_mesh_router#50] [Q&A]
Hello good morning, I installed your integration and activated the advanced mode but I cannot use the advanced functions such as blacklist and whitelist. Would you help me?

I am an enthusiast, I still have many difficulties. Is there a video of you explaining better the details of how to use this integration?

![Captura de Tela (2)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/fe6b0b5b-2670-41a6-bf80-3d31c065aeb8)
![Captura de Tela (3)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/037d3755-3120-4732-92d5-93f968f2a7f3)

**Comment by vmakeev** [comment]
Hello!

At first glance, it seems that the component believes that the function is not supported by your router.

To begin with, let's try to restart the integration, perhaps it was a one-time failure

![image](https://github.com/vmakeev/huawei_mesh_router/assets/13693034/7b0dfd26-a343-44da-ac27-20e67a0c340e)


Try to go to the wi-fi access restrictions page in the web interface of your router (`http://<your-router-ip>/html/index.html#/more/wlanaccess`), is it available? The page should look something like this:

![image](https://github.com/vmakeev/huawei_mesh_router/assets/13693034/5273f024-7c6b-4d3b-aea6-4571e80b7678)

**Comment by vmakeev** [comment]
@edinhodias7 has reloading the integration solved the problem? Please let me know

**Reply by edinhodias7** [reply]
Hello, I did as you described me, but it remains the same. Thank you for your attention.
![Captura de Tela (6)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/8e9c4570-e99b-4c6f-8ef6-7a8823e65ed2)
![Captura de Tela (4)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/b5eaadcb-c4b0-494a-93f6-ef04d776af0a)
![Captura de Tela (5)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/542618c5-7845-4d92-9a7c-759aaa9b32d2)
![Captura de Tela (7)](https://github.com/vmakeev/huawei_mesh_router/assets/83982081/86b7d1a0-5f87-4ab7-ac9f-909bed24bac8)
